packages/clerk-js/src/ui/signUp/SignUpStart.tsx

```diff
@@ -153,7 +159,9 @@ function _SignUpStart(): JSX.Element {
 
     try {
       setError(undefined);
-      const res = await signUp.create(buildRequest(reqFields));
+      const req = buildRequest(reqFields);
+      const promise = continueExisting ? signUp.update(req) : signUp.create(req);
```

Comment:
The component should be smart enough to pick between create or update if signUp has an id. I'd also suggest to introduce a signUp.save method on the resource model that will handle this conditional.

Reply:
You can have a previous sign up and this component with create a new one anyway (as it works now).

So currently it's not a matter of having an id.

Do we want to change this?

Reply:
Hmmm, you are right. We always create a new sign up. When will the user land on this component with an existing sign up? Won't the `<SignUp/>` redirect them to `<SignUpContinue/>`?

Reply:
Correct. I believe it's more clear this way.